flink-dist/src/main/flink-bin/bin/migrate-config-tool.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+USAGE="Usage: migrate-config-tool.sh FLINK_CONF_DIR FLINK_BIN_DIR FLINK_LIB_DIR"
```

Comment:
I prefer to name it as `migrate-config-file.sh`.

`FLINK_CONF_DIR` and `FLINK_BIN_DIR` and `FLINK_LIB_DIR` should be automatically decided for ease of use.
Optionally, we may make it to accept the legacy file path as a param.